modules/administration/pages/ssl-certs.adoc

```diff
@@ -1,9 +1,17 @@
-[[custom-ssl]]
-= Custom SSL Certificates
+[[ssl-certs]]
+= SSL Certificates
 
-You can use custom certificates with {productname} and {productname} Proxy.
+{productname} uses SSL certificates to ensure that clients are registered to the correct server.
 
-This section covers how to use a third party SSL certificate authority with a new {productname} installation, and replacing existing certificate with new custom certificates.
+When you create an SSL certificate for your server, the certificate includes the host name of the server it is associated with.
+The certificate is then deployed on the server, and when clients attempt to connect, they check the certificate details.
+If the certificate contains the name of the server they are attempting to connect to, the connection will be successful.
+The client will also check if the certificate has been signed by a trusted certificate authority.
+
+By default, {productname} uses a self-signed certificate.
```

Comment:
Found this while google for `self-signed certificate`:

`In cryptography and computer security, a self-signed certificate is a certificate that is not signed by a certificate authority (CA). `

This means SUSE Manager is **not** using this. We generate a CA and sign our server certificate with it.

Reply:
But we're not a certificate authority ... ?

Reply:
Seems I understood self-signed different to what people outside think it is. Maybe you can verify with wikipedia, etc. if there is a common understanding out there.

For me, self-signed is a certificate which is signed by itself and is at the same time CA and server certificate. This is something we cannot use at SUSE Manager.

But it seems that people also understand self-signed, as signed by an own PKI. You created your CA and use it to sign your own certificates with it. This is ok, and this is what we also do in the SUSE Manager setup procedure.
So here you are your own CA. 

So we need to differ between "internally" and "externally" generated certificates.

Reply:
OK, let me figure this out.

Reply:
I suspect the problem stems from definitions of a CA, not from definitions of a self-signed certificate. If you define a CA as any organisation that signs certificates, it's a bit murky. If a CA is an organisation that will perform research and verify a certificate for *other* organisations, then the difference becomes clear. A self-signed certificate is a certificate signed by the organisation that intends to use it (one website I looked at suggested that it was the same as approving your own passport application, which is oversimplified, but a good visual), and not by an external CA. Either way, the certificate must be signed, it's just a question of who does the signing, and under what authority.

Reply:
So now I would like to know what we mean when we say "custom" certificate, because the way we've used the term in this doc makes me think it's neither a self-signed cert, or one verified by a CA.

Reply:
And once I have the various certificate types clear in my head, I'm going to split this chapter, I think.

Reply:
I think we need to distinguish between certificates generated by our tools or getting files from external. That external can be an own PKI from the infra team or a third party CA. But this does not matter in our case. 

We either generate the certs with out tools or we get the certificates and keys from somewhere else.

Now the question is: how to name this properly

Reply:
Maybe: self generated(or SUSE Manager generated) vs. imported ?